docusaurus.config.js

```diff
@@ -33,7 +33,7 @@ const config = {
           sidebarPath: require.resolve("./sidebars.js"),
           routeBasePath: "/",
           // Change this to your repo.
-          editUrl: "https://github.com/Green-Software-Foundation/ief/",
+          editUrl: "https://github.com/Green-Software-Foundation/if-docs/edit/master",
```

Comment:
```suggestion
          editUrl: "https://github.com/Green-Software-Foundation/if-docs/blob/master/docs/",
```
I think this is the right slug

Reply:
@jmcook1186 Thanks for the suggestion! However, the original editUrl provided before the suggestion is actually the correct one. It properly points to the repository where the documentation can be edited. The inclusion of docs in the URL is not needed in this case as the base repository is already the correct location for edits.